src/sdk/main/src/TokenInfo.cc

```diff
@@ -288,6 +297,17 @@ std::string TokenInfo::toString() const
   }
 
   json["mLedgerId"] = mLedgerId.toString();
+
+  if (!mMetadata.empty())
+  {
+    json["mMetadata"] = internal::HexConverter::bytesToHex(mMetadata);
+  }
+
+  if (mMetadataKey)
+  {
+    json["mMetadataKey"] = internal::HexConverter::bytesToHex(mMetadataKey->toBytes());
```

Comment:
```suggestion
    json["mMetadataKey"] = mMetadataKey->toString();
```

Reply:
Was there a reason you didn't use `toString()` here instead?

Reply:
I think here we only have to `toProtobuf` and `toBytes` provided by the `Key` interface, `toString` gets introduced in the inherited keys `ECDSA`,`ED25519`.